mysys/my_atomic_writes.c

```diff
@@ -121,8 +121,10 @@ static my_bool test_if_shannon_card_exists()
     }
 #endif
     shannon_devices[shannon_found_devices].st_dev= stat_buff.st_rdev;
-    sprintf(shannon_devices[shannon_found_devices].dev_name, "/dev/sct%c",
-            dev_part);
+    snprintf(shannon_devices[shannon_found_devices].dev_name,
```

Comment:
Likewise enough space for this fixed size is already allocated, `snprintf` is redundant.